ee/agent/internal/informer_modules.go

```diff
@@ -109,15 +105,10 @@ func NewModuleEventHandler(logger logging.Logger, membershipClient MembershipCli
 			oldStatus, err := getStatus(oldVersions)
 			if err != nil {
 				logger.Errorf("Unable to get status from old versions: %s", err)
-				return
 			}
-			newStatus, err2 := getStatus(newVersions)
-			if err2 != nil {
-				logger.Errorf("Unable to get status from new versions: %s", err2)
-				return
-			}
-
-			if newStatus == nil {
+			newStatus, err := getStatus(newVersions)
+			if err != nil {
+				logger.Errorf("Unable to get status from new versions: %s", err)
 				return
 			}
```

Comment:
We want to continue the process even if there is no `status` defined on oldObj. Netherless, it has to be set on `newObj`